services/sharepoint-connector/src/processing-pipeline/steps/content-registration.step.ts

```diff
@@ -22,43 +17,34 @@ import type { IPipelineStep } from './pipeline-step.interface';
 export class ContentRegistrationStep implements IPipelineStep {
   private readonly logger = new Logger(this.constructor.name);
   public readonly stepName = PipelineStep.ContentRegistration;
-  private readonly ingestionMode: IngestionMode;
-  private readonly scopeId: string | undefined;
-  private readonly rootScopeName: string | undefined;
   private readonly sharepointBaseUrl: string;
 
   public constructor(
     private readonly uniqueFileIngestionService: UniqueFileIngestionService,
     private readonly configService: ConfigService<Config, true>,
   ) {
-    this.ingestionMode = this.configService.get('unique.ingestionMode', { infer: true });
-    this.scopeId = this.configService.get('unique.scopeId', { infer: true });
-    this.rootScopeName = this.configService.get('unique.rootScopeName', { infer: true });
     this.sharepointBaseUrl = this.configService.get('sharepoint.baseUrl', { infer: true });
   }
 
   public async execute(context: ProcessingContext): Promise<ProcessingContext> {
     const stepStartTime = Date.now();
 
-    const scopeId = getScopeIdForIngestion(this.ingestionMode, this.scopeId);
-
     const itemKey = buildIngestionItemKey(context.pipelineItem);
-    const baseUrl = this.rootScopeName || this.sharepointBaseUrl;
 
     const contentRegistrationRequest: ContentRegistrationRequest = {
       key: itemKey,
       title: context.pipelineItem.fileName,
       mimeType: context.mimeType ?? DEFAULT_MIME_TYPE,
       ownerType: UniqueOwnerType.Scope,
-      scopeId: scopeId,
+      scopeId: context.scopeId,
       sourceOwnerType: UniqueOwnerType.Company,
       sourceKind: INGESTION_SOURCE_KIND,
       sourceName: INGESTION_SOURCE_NAME,
-      ...(this.ingestionMode === IngestionMode.Recursive && {
-        url: context.knowledgeBaseUrl,
-        baseUrl,
-      }),
+      url: context.knowledgeBaseUrl,
+      baseUrl: this.sharepointBaseUrl,
+      byteSize: context.fileSize ?? 0,
```

Comment:
I am always passing now byteSize because something changed in other systems and files with 0 byteSize are no longer displayed in knowledge base.